### PR TITLE
Preparation for release v2024.1 (Backend)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Preparation for release.

## Changes

Updates files `Changes`, `Backend.pm` and `Makefile.PL`.

## How to test this PR

Review the changes.

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+v11.2.0 2024-07-01 (part of Zonemaster v2024.1 release)
+
+ [Release information]
+ - Translations have not been fully updated in this release. They will
+   be updated in an upcoming release.
+ - Migration of the database is required by this release to retain
+   consistency in results in the database.
+
+ [Features]
+ - Ensures Backend’s messages get translated (#1166)
```
Comment (marc-vanderwal) [inline Changes:13]: Bringing in line with the appropriate PR’s title (see #1166):
```suggestion
 - Ensure Backend’s messages stay translated (#1166)
```

File: Changes [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+
+ [Fixes]
+ - Avoids double UTF-8 encoding in zmb (#1144)
+ - Makes job_status report proper percentages (#1150)
+ - Updates language in experimental RPC API paramaters (#1155)
```
Comment (marc-vanderwal) [inline Changes:18]:
```suggestion
 - Updates language in experimental RPC API parameters (#1155)
```

**Review by MichaelTimbert — DISMISSED** [review]

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes:13]: The bullet always starts with a verb in passive form. The changes "ensures ...".

Comment (matsduf) [reply Changes:13]: Doesn't "get translated" cover that? Nothing stays translated.

Comment (matsduf) [reply Changes:13]: "are translated"?

Comment (matsduf) [reply Changes:13]: In general, unit tests are not covered by the Changes file. We agreed on that some years back. Travis changes are not included either.

Comment (marc-vanderwal) [reply Changes:13]: In that case, how about:
```suggestion
 - Fixes regression that caused Backend’s messages not to be translated (#1166)
```

**Review by marc-vanderwal — APPROVED** [review]

**Comment by matsduf** [comment]
@marc-vanderwal @MichaelTimbert @mattias-p @tgreenx Please re-review.

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes]: Fixed.

**Pull request merged**